### ISSUE TITLE
Should the Brand Icon Browser be its own repository?

### ISSUE THREAD

**Issue by quamsta** [uiowa/brand-icons#10]
Looking for some feedback from @uiowa/developer, specifically @pyrello if you have any thoughts on this:

- Should the `brand-icon` [image files](https://github.com/uiowa/brand-icons/tree/brand-icon-browser/public/brand-icons) and [json data](https://github.com/uiowa/brand-icons/tree/brand-icon-browser/data) live separately as a repository and/or node package that can be required in a theoretical `brand-icon-browser` repository?

My thinking behind this is that keeping those two parts of this project separate would make it easier for the design team to edit and push up changes while keeping the VueJS app development separate and allow for easier versioning of icons/building new versions of the app. 

To summarize:

- `brand-icons` would contain [image files](https://github.com/uiowa/brand-icons/tree/brand-icon-browser/public/brand-icons) and [json data](https://github.com/uiowa/brand-icons/tree/brand-icon-browser/data) 
- `brand-icon-browser` would contain the VueJS

Looking for feedback on this. It may just be making things too complicated.

**Comment by pyrello** [comment]
In support of the idea of splitting this into two repos, I think that would decrease the likelihood of issues related to brand team having to navigate around the Vue.js app structure.

On the other hand, having two repos means that adding an icon to the browser will require PRs in both repositories. One in the `brand-icons` repo to actually add the new icon, and one in the `brand-icon-browser` to update the version of the `brand-icon` package being used. We encounter this with UIDS and have considered combining that project into `uiowa` to get around it (we decided against it for now).

I wonder about the possibility of using git submodules? I don't know much about them, but understand they are a way to embed one repo in another one. I'm not sure if they would allow us to avoid the above issue or not.

**Comment by quamsta** [comment]
In my experience in the distant past, git submodules are finicky and present some strange behavior. However it's been a while since I've used them and GitHub may have cleaned up the way they present and implement them.

Another idea I had if we kept them separated is that the `brand-icons` image files and json data could be presented as endpoint(s) that the VueJS app could consume dynamically, so that when `brand-icons` is updated and merged into a versioned branch, those image and json files could be deployed to a server and the VueJS app would just consume them without needing to be updated or touched at all.

**Comment by quamsta** [comment]
I think we should probably split the icon browser off to a separate repo very soon, with its own set of icon files and data (for now). 

It might be good to do this early on so we can start making issues for the browser specifically as folks start code reviewing it. I'd prefer this over a giant to-do list issue on this repo. @bspeare, cool with a new repo getting added?

**Comment by quamsta** [comment]
New repository here:

https://github.com/uiowa/brand-icon-browser
